Xamarin.PropertyEditing.Mac/Controls/Variations/CreateVariantView.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using AppKit;
+using CoreGraphics;
+using Foundation;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class CreateVariantView
+		: NSView
+	{
+		private const int ControlSpacing = 7;
+		private const int HorizontalControlSpacing = 10;
+		internal const int RightEdgeMargin = 12;
+
+		private readonly IHostResourceProvider hostResources;
+
+		public CreateVariantView (IHostResourceProvider hostResources, CGSize windowWidthHeight, CreateVariantViewModel createVariantViewModel)
+		{
+			if (hostResources == null)
+				throw new ArgumentNullException (nameof (hostResources));
+
+			this.hostResources = hostResources;
+
+			if (createVariantViewModel == null)
+				throw new ArgumentNullException (nameof (createVariantViewModel));
+
+			Initialize (createVariantViewModel, windowWidthHeight);
+		}
+
+		private void Initialize (CreateVariantViewModel createVariantViewModel, CGSize windowWidthHeight)
+		{
+			Frame = new CGRect (CGPoint.Empty, windowWidthHeight);
+
+			int editorHeight = 18;
+
+			var FrameWidthThird = Frame.Width / 3;
```

Comment:
Shouldn't this be the same sizing as the panel?

Reply:
The idea was that the VariationCategory Name would be take up the left 3rd of the initial window, while the Category drop downs would take up the remaining 2/3rds and expand.
As per...
<img width="251" alt="Screen Shot 2019-10-02 at 10 06 52" src="https://user-images.githubusercontent.com/271363/66032064-97222680-e4fc-11e9-80a3-81e4261c8dec.png">
I did think of making it follow Golden ratio resizing, as per our property list, but thought it might be overkill of this popup window.